### ISSUE TITLE
How to use multi-GPU training

### ISSUE THREAD

**Issue by CSU-NXY** [google-research/multinerf#36]
Hi, I'm trying to train MipNeRF360 with 4 GPUs. Should I remain the batch_size untouched or multiply 4 to it? Is there any other changes I should do to support multi-GPU training? 

Thanks!

**Comment by dogyoonlee** [comment]
@CSU-NXY 
Did you solved?
if it is solved, please share the solution!
Thank you

**Comment by jonbarron** [comment]
If you decrease the batch size, you should increase the number of iterations and decrease the learning rate according to the linear scaling rule.

**Comment by hecodeit** [comment]
@CSU-NXY I'm same question as you, did you find the way of configuration for multiple GPU?
